integration/hook-useSubmit-test.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from "@playwright/test";
+
+import { createAppFixture, createFixture, js } from "./helpers/create-fixture";
+import type { Fixture, AppFixture } from "./helpers/create-fixture";
+import { PlaywrightFixture } from "./helpers/playwright-fixture";
+
+test.describe("`useSubmit()` returned function", () => {
+  let fixture: Fixture;
+  let appFixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await createFixture({
+      files: {
+        "app/routes/index.jsx": js`
+        import { useLoaderData, useSubmit } from "@remix-run/react";
+
+        export function loader({ request }) {
+          let url = new URL(request.url);
+          return url.searchParams.toString()
+        }
+
+        export default function Index() {
+          let submit = useSubmit();
+          let handleClick = event => submit(nativeEvent.submitter || e.currentTarget)
```

Comment:
Where does `nativeEvent` come from?

Reply:
Oups, thanks. I've fixed that. The test was passing because the `<form>` was natively submitting with the GET method.

Reply:
Amended again to call `event.preventDefault()` to only have the behavior of `submit()` being tested. I've double-checked the test and it now test what it meant to.